### PR TITLE
Added numbers to show ATP balance

### LINKED ISSUE #1559: Add numbers back to the ATP balance bar
to at least show the totals

### PR DESCRIPTION
**Pull request by DTanxxx**
Added energyBalance.FinalBalance to the label's text.

Closes #1559

### CODE REVIEW

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
This is not what I imagined the bars should do.
Instead both the production and consumption bar should show a number for their details.
Just showing the total balance doesn't solve the issue "Add numbers **back** to the ATP --" which means that before both the bars showed their total value, and they don't anymore and it'd be nice to have that back.

**Comment by DTanxxx** [comment]
Oh right, is it fine if I add labels to both bars showing the totals (since you said before that it would look cluttered)?

**Comment by hhyyrylainen** [comment]
That's what I want. I meant that as the bars (there are 2) have sub-bars (these are the ones with icons and different colours), having numbers on those sub-bars would make things cluttered. I do want both of the bars to have their totals on them.

**Comment by DTanxxx** [comment]
I added labels underneath the two bars so it's easier to read.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
The numbers should be to the right of the bars in order to not need the explanation text that takes up extra space.

File: src/microbe_stage/editor/MicrobeEditorGUI.cs [hunk 1]
```diff
@@ -460,6 +470,12 @@ public void UpdateEnergyBalance(EnergyBalanceInfo energyBalance)
             atpBalanceLabel.AddColorOverride("font_color", new Color(1.0f, 0.2f, 0.2f));
         }
 
+        atpProductionLabel.Text = $"{energyBalance.TotalProduction}";
+        atpProductionLabel.AddColorOverride("font_color", new Color(1.0f, 1.0f, 1.0f));
```
Comment (hhyyrylainen) [inline src/microbe_stage/editor/MicrobeEditorGUI.cs:474]: Is this needed? Isn't this the default colour?

**Comment by hhyyrylainen** [comment]
There isn't much code to review, so I requested @Kasterisk reviews the GUI side of things.

File: src/microbe_stage/editor/MicrobeEditorGUI.cs [hunk 2]
```diff
@@ -460,6 +470,12 @@ public void UpdateEnergyBalance(EnergyBalanceInfo energyBalance)
             atpBalanceLabel.AddColorOverride("font_color", new Color(1.0f, 0.2f, 0.2f));
         }
 
+        atpProductionLabel.Text = $"{energyBalance.TotalProduction}";
```
Comment (Kasterisk) [inline src/microbe_stage/editor/MicrobeEditorGUI.cs:473]: Shoud be formatted to as how it was before: `string.Format(CultureInfo.CurrentCulture, "{0:F1}", energyBalance.TotalProduction);` (and the same for `TotalConsumption`) to limit the decimals to only one digit

**Review by Kasterisk — CHANGES_REQUESTED** [review]
For the labels, the horizontal `fill` size flag also needs to be enabled so the text's horizontal alignment can take effect.

File: project.godot [hunk 3]
```diff
@@ -154,13 +154,13 @@ e_rotate_left={
 }
 e_undo={
 "deadzone": 0.5,
-"events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":true,"meta":false,"command":true,"pressed":false,"scancode":90,"unicode":0,"echo":false,"script":null)
+"events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":true,"meta":true,"command":true,"pressed":false,"scancode":90,"unicode":0,"echo":false,"script":null)
```
Comment (hhyyrylainen) [inline project.godot:157]: These are probably once again unintended keybinding changes.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

**Review by Kasterisk — APPROVED** [review]
Other than hhyyrylainen mentioned above, looks good

**Review by hhyyrylainen — APPROVED** [review]

**Comment by hhyyrylainen** [comment]
Hey @DTanxxx you have done quite a few PRs now. Have you considered applying to the team https://revolutionarygamesstudio.com/application/ ? You can use your PRs as work samples to get accepted.

**Pull request merged**